lib/gtfsplus/components/GtfsPlusEditor.js

```diff
@@ -83,23 +86,25 @@ export default class GtfsPlusEditor extends Component {
 
   render () {
     const {
+      activeTableId,
       feedSource,
-      deleteRowClicked,
-      fieldEdited,
-      newRowClicked,
       project,
       tableData,
       user,
-      validation
+      validation,
+      visibleRows
     } = this.props
     if (!feedSource) return null
     const editingIsDisabled = !user.permissions.hasFeedPermission(feedSource.organizationId, feedSource.projectId, feedSource.id, 'edit-gtfs')
     const buttonStyle = {
       display: 'block',
-      width: '100%'
+      width: '100%',
```

Comment:
Static styling should live in css files.

Reply:
I find that putting it in css can be confusing.  Why not just keep it in the js?

Reply:
https://www.ctheu.com/2015/08/17/react-inline-styles-vs-css-stupid-benchmark/